src/web/components/Home/DocumentationCard.tsx

```diff
@@ -3,7 +3,7 @@ import './DocumentationCard.scss';
 function DocumentationCard() {
   return (
     <div className='document-card'>
-      <img src='/document.svg' alt='document' />
+      <img src='/document.svg' alt='document' width='50px' />
```

Comment:
nit: perhaps this width styling should live in `DocumentationCard.scss` for consistency?

Reply:
Usually I'd agree, but I think because the size is tied to the image file (which is referenced here), it should be here. I'd like to find a plugin to automatically add these at some point.